### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by emailboxu**
4260 - Korean has 3 separate sentences, combined them into 2 for continuity.

4281 - 두더지 -> Mole? Or just romanize it? 

4282 - 두더비 -> What is this? Currently replaced with a romanization.

~4300.

### CODE REVIEW

File: ITEM_en.tsv [hunk 1]
```diff
@@ -4199,104 +4199,104 @@ ITEM_20150406_004575	A plow for ploughing fields. It has a keen blade.
...
+ITEM_20150414_004625	5kg Dumbbell
+ITEM_20150414_004626	When you pop the bubbles, you feel better.
+ITEM_20150414_004627	Defeated Flag
+ITEM_20150414_004628	A mushroom with a hint of a woody scent.
 ITEM_20150414_004629	White Bread
-ITEM_20150414_004630	It is a soft bread. It taste better if spread the jam on the bread.
+ITEM_20150414_004630	A soft, white bread. It tastes better if jam is spread on it.
 ITEM_20150414_004631	Lettuce
-ITEM_20150414_004632	A crunched vegetable with good taste.
+ITEM_20150414_004632	A vegetable with a crunchy testure.
```
Comment (megaoka) [inline ITEM_en.tsv]: testure -> texture

File: ITEM_en.tsv [hunk 2]
```diff
@@ -4199,104 +4199,104 @@ ITEM_20150406_004575	A plow for ploughing fields. It has a keen blade.
...
-ITEM_20150414_004616	Don't be deliberately damaged even you want to check the magic defense.
-ITEM_20150414_004617	Gloves Keeper
-ITEM_20150414_004618	Equipment used by Keeper
-ITEM_20150414_004619	Boots Keeper
-ITEM_20150414_004620	Pants Keeper
-ITEM_20150414_004621	Tunic Keeper
+ITEM_20150414_004615	Cotton Robes
+ITEM_20150414_004616	Don't test this armor's Magic Defense by intentionally being hit.
+ITEM_20150414_004617	Watchman Gloves
+ITEM_20150414_004618	Equipment used by Watchmen.
```
Comment (megaoka) [inline ITEM_en.tsv]: For the description: Watchmen -> watchmen ?

I'm assuming these are just generic guards/watchmen, unless they need to be capitalized here.

**Comment by vegax87** [comment]
**CORRECT**: ITEM_20150406_004579   Well dried beef jerky.
**WRONG**: ITEM_20150406_004579 Farm equipment for separating fields. It has a lot of sharp points.

maybe you haven't updated your fork

**Comment by emailboxu** [comment]
Nope... looking at the master Korean translation, https://github.com/Treeofsavior/EnglishTranslation/blob/master/ITEM.tsv, not a fork..

ITEM_20150406_004579    밭을 고르는데 쓰는 농기구. 뾰족한 곳이 많다.

File: ITEM_en.tsv [hunk 3]
```diff
@@ -4199,104 +4199,104 @@ ITEM_20150406_004575	A plow for ploughing fields. It has a keen blade.
...
 ITEM_20150414_004613	Cotton Boots
 ITEM_20150414_004614	Cotton Pants
-ITEM_20150414_004615	Cotton Robe
-ITEM_20150414_004616	Don't be deliberately damaged even you want to check the magic defense.
-ITEM_20150414_004617	Gloves Keeper
-ITEM_20150414_004618	Equipment used by Keeper
-ITEM_20150414_004619	Boots Keeper
-ITEM_20150414_004620	Pants Keeper
-ITEM_20150414_004621	Tunic Keeper
+ITEM_20150414_004615	Cotton Robes
```
Comment (Sunaries) [inline ITEM_en.tsv:4228]: Robes -> Robe 

It's not plural.

**Comment by imcgames** [comment]
I think "두더비" is a typo instead of "두더지". :)
Thank you :+1:

**Pull request merged**